code/modules/spells/spell_types/touch/scream_for_me.dm

```diff
@@ -16,20 +16,49 @@
 	hand_path = /obj/item/melee/touch_attack/scream_for_me
 
 /datum/action/cooldown/spell/touch/scream_for_me/on_antimagic_triggered(obj/item/melee/touch_attack/hand, mob/living/victim, mob/living/carbon/caster)
-	caster.visible_message(
-		span_warning("The feedback mutilates [caster]'s arm!"),
-		span_userdanger("The spell bounces from [victim]'s skin back into your arm!"),
+
+
+	// Informs the wizard of the backfire
+	to_chat(victim, span_warning("The feedback mutilates [caster]'s arm!"))
+	to_chat(caster, span_userdanger("The spell bounces from [victim]'s skin back into your arm!"))
+
+	var/list/wounds = list(
+		WOUND_SEVERITY_TRIVIAL,
+		WOUND_SEVERITY_MODERATE,
+		WOUND_SEVERITY_SEVERE,
 	)
+
+	// Gets the holding hand of the touch attack.
 	var/obj/item/bodypart/to_wound = caster.get_holding_bodypart_of_item(hand)
-	caster.cause_wound_of_type_and_severity(WOUND_SLASH, to_wound, WOUND_SEVERITY_MODERATE, WOUND_SEVERITY_CRITICAL)
+
+	// Wounds the bodypart severely.
+	caster.cause_wound_of_type_and_severity(WOUND_SLASH, to_wound, pick(wounds))
```

Comment:
This is changing the previous behavior that moderate is meant to be the minimum and critical is the max, as per the args of ``cause_wound_of_type_and_severity``. What's the change that's fixing the ability?